Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
     encryptor (3.0.0)
     erubi (1.12.0)
     erubis (2.7.0)
-    execjs (2.7.0)
-    faraday (2.7.11)
+    execjs (2.9.1)
+    faraday (2.8.1)
       base64
       faraday-net_http (>= 2.0, < 3.1)
       ruby2_keywords (>= 0.0.4)
-    faraday-http-cache (2.5.0)
+    faraday-http-cache (2.5.1)
       faraday (>= 0.8)
     faraday-multipart (1.0.4)
       multipart-post (~> 2)
     faraday-net_http (3.0.2)
     faraday-net_http_persistent (2.1.0)
       faraday (~> 2.5)
       net-http-persistent (~> 4.0)
-    ffi (1.15.5)
+    ffi (1.16.3)
```

Comment:
### [ffi](https://github.com/ffi/ffi/wiki)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.15.5 → 1.16.3

[_[change-log](https://github.com/ffi/ffi/blob/master/CHANGELOG.md), [source-code](https://github.com/ffi/ffi/), [gem-diff](https://my.diffend.io/gems/ffi/1.15.5/1.16.3)_]





<details>
<summary>Commits</summary>

A change of **183** commits. See the full changes on [the compare page](https://github.com/ffi/ffi/compare/v1.15.5...v1.16.3).

These are the first 10 commits:
- (5b459e7) [Fix task dependencies in Rakefile](https://github.com/ffi/ffi/commit/5b459e752c9b4f0d3846681537c79d0cc158b8a7)
- (3034402) [CI: Fix appveyor build](https://github.com/ffi/ffi/commit/30344025ff02e9620b71498cc2e7b6b8afdb9782)
- (0c15d7b) [Use recommended rake tasks for cross build](https://github.com/ffi/ffi/commit/0c15d7baf0d8928c5e5cc69e11eb50b2205ee123)
- (447845c) [Bump rake-compiler to ~>1.1](https://github.com/ffi/ffi/commit/447845cb3030194c79700c86fb388a12e6f81386)
- (5153384) [Add types.conf for loongarch64-linux](https://github.com/ffi/ffi/commit/5153384b330ccf1ecf07a98cbc5ea1a291f6059c)
- (edbee9d) [Fix typo.](https://github.com/ffi/ffi/commit/edbee9ded2bfdb3de19d46137ffe6e2b85f5d350)
- (bdb4ede) [Remove unneeded affectations.](https://github.com/ffi/ffi/commit/bdb4edeae927b1bf4d867175c21d110c17812fbd)
- (5f71e6f) [Simplify from_native and test corner case.](https://github.com/ffi/ffi/commit/5f71e6f9624e00bc0618005d5593a0b422df73b7)
- (e7ed8da) [Fix an issue with signed bitmasks when using flags on the mo…](https://github.com/ffi/ffi/commit/e7ed8da894f30e67384f20fb6e1c909b1fc5d6e9)
- (c85c406) [Skip problematic test involving TruffleRuby.](https://github.com/ffi/ffi/commit/c85c406e383419764b59b55e4b27faf99f817855)


</details>